db/index.js

```diff
@@ -127,6 +129,60 @@ function create(options) {
     })
   }
 
+  var timerIdTtlScanner = null
+  if (typeof options.ttlCheckEvery === 'number'  && options.ttlCheckEvery > 0) {
+    var ttlScannerInterval = options.ttlCheckEvery * 1000
+
+    timerIdTtlScanner = setInterval(function () {
+      var currentUnixSeconds = Math.round(Date.now() / 1000)
+
+      function logError(err, result) {
+        if (err) console.error("@@@", err)
+      }
+
+      lazyStream(tableDb.createKeyStream({}), logError)
```

Comment:
This may be a completely ignorant comment but is it possible this is an inefficient way to query our table names?

Overall I'm very curious as to how expensive this operation is to run. It's probably not likely to blow up a modern machine, but it's O(n^lots). I wonder if a more efficient approach would be to create an internal ttl index and query that instead of streaming out + operating on the contents of each table.

Reply:
I do not have time to create such an index for efficient traversing.